testing/jormungandr-testing-utils/resources/explorer/graphql/schema.graphql

```diff
@@ -1,6 +1,146 @@
 """Directs the executor to query only when the field exists."""
 directive @ifdef on FIELD
 
+type Block0Date {
+  block0_date: Int!
+}
+
+enum DiscriminationEnum {
+  PRODUCTION,
+  TEST,
+}
+
+type Discrimination {
+  discrimination: DiscriminationEnum!
+}
+
+enum ConsensusTypeEnum {
+  BFT,
+  GENESISPRAOS,
+}
+
+type ConsensusType {
+  consensus_type: ConsensusTypeEnum!
+}
+
+type SlotsPerEpoch {
+  slots_per_epoch: Int!
+}
+
+type SlotDuration {
+  slot_duration: Int!
+}
+
+type EpochStabilityDepth {
+  epoch_stability_depth: Int!
+}
+
+type Milli {
+  milli: Int!
+}
+
+type BlockContentMaxSize {
+  block_content_max_size: Int!
+}
+
+type AddBftLeader {
+  add_bft_leader: BftLeader!
+}
+
+type RemoveBftLeader {
+  remove_bft_leader: BftLeader!
+}
+
+type PerCertificateFee {
+  certificate_pool_registration: NonZero
+  certificate_stake_delegation: NonZero
+  certificate_owner_stake_delegation: NonZero
+}
+
+type PerVoteCertificateFee {
+  certificate_vote_plan: NonZero
+  certificate_vote_cast: NonZero
+}
+
+type LinearFee {
+  constant: Int!
+  coefficient: Int!
+  certificate: Int!
+  per_certificate_fees: PerCertificateFee!
+  per_vote_certificate_fees: PerVoteCertificateFee!
+}
+
+type ProposalExpiration {
+  proposal_expiration: Int!
+}
+
+type KesUpdateSpeed {
+  kes_update_speed: Int!
+}
+
+type TreasuryAdd {
+  treasury_add: Value!
+}
+
+type TreasuryParams {
+  treasury_params: TaxType!
+}
+
+type RewardPot {
+  reward_pot: Value!
+}
+
+type Linear {
```

Comment:
given that graphql doesn't have modules (as far as I know), I think I would call this `LinearRewardParams`. If you see `Linear` in the schema then it's really not that obvious what it is.